build.sbt

```diff
@@ -43,3 +55,17 @@ lazy val importer = project
   .in(file("modules/importer"))
   .dependsOn(core, db)
   .settings(commonSettings)
+
+lazy val json = project
+  .in(file("modules/json"))
+  .dependsOn(core)
+  .settings(commonSettings)
+  .settings(
+    libraryDependencies ++= Seq(
+      "io.argonaut"                %% "argonaut"               % argonautVersion,
```

Comment:
Ok, good to know. I've been thinking to support binary and json on the web front end. http4s leans to circe but we have mostly used argonaut in the past